lib/ember-cli/asset_resolver.rb

```diff
@@ -0,0 +1,53 @@
+module EmberCLI
+  class AssetResolver
+    def initialize(options)
```

Comment:
I agree, support for older Rails versions should be dropped.

The same could probably be said for Ruby versions.

Since I've only recently inherited maintainership of this project, I'll try not to rock the boat.

In the future I'll make some sort of announcement specifying version boundaries.